spec/lib/doi_search_spec.rb

```diff
@@ -23,7 +23,7 @@
           provenance: 'cap',
           identifier: [{ type: 'doi', id: '10.1111/j.1444-0938.2010.00524.x', url: 'https://dx.doi.org/10.1111/j.1444-0938.2010.00524.x' }]
         }
-        publication.sync_identifiers_in_pub_hash_to_db
+        publication.send(:sync_identifiers_in_pub_hash_to_db)
         publication.save
```

Comment:
Rhetorical Q:  So does this mean it's a public method?  Maybe not, it's only in a spec.  No problem.

Reply:
Nothing should be public just for the purpose of calling it in tests.

Reply:
Yes, agreed.  In this case, it's just a little bit odd that the spec is on a different class.  That's what prompted a bit of extra thought and a comment to myself there, like Ariadne's thread.  Perhaps I can make a pattern out of that -- if I prefix a comment with `Rhetorical Q:` or `Note to self:` then it's just my way of noting that I've looked at it thought about and arrived at a conclusion myself.